docs/install/openshift.md

```diff
@@ -0,0 +1,263 @@
+## Requirements
+
+Before proceeding, please ensure that you have an OpenShift cluster running K8s
+1.19+ (OpenShift 4.7+) and have Helm 3.5+ installed. In addition, you'll need to
+install the OpenShift CLI (`oc`) to authenticate to your cluster and create OpenShift
+resources.
+
+You'll also want to install the [latest version of Coder](https://github.com/coder/coder/releases/latest)
+locally in order to log in and manage templates.
+
+## Install Coder with OpenShift
+
+### 1. Authenticate to OpenShift and create a Coder project
+
+Run the following command to login to your OpenShift cluster:
+
+```console
+oc login <cluster-url>
+```
```

Comment:
this just returns a message that you need an API token.

suggest we replace with:

oc login --token=sha256~<token> --server=<cluster-url>

Reply:
@sharkymark yes but the `oc login <cluster-url` flow gets the user to retrieve the token